pom.xml

```diff
@@ -236,20 +326,39 @@
           <artifactId>maven-compiler-plugin</artifactId>
           <version>3.6.1</version>
         </plugin>
+
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-war-plugin</artifactId>
           <version>2.6</version>
         </plugin>
+
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-resources-plugin</artifactId>
+          <version>3.0.2</version>
+        </plugin>
```

Comment:
Can this be removed as discussed?

Reply:
Thanks for catching this; I modified it so it doesn't require `maven-resources-plugin` anymore, but forgot to remove it from `dependencyManagement`.